articles/implementing-network-address-translation/index.md

```diff
@@ -0,0 +1,205 @@
+# Implementing Network Address Translation (NAT) on a CISCO router
+![hero Image](/engineering-education/implementing-network-address-translation/hero.jpg)
+[source](https://unsplash.com/photos/eVWWr6nmDf8)
+### Introduction
+This tutorial will cover how to implement NAT on a CISCO router, and it is expected that at the of the tutorial, the reader should learn the following:
+- What Network Address Translation is
+- Types of NAT
+- How to configure different types of NAT on a CISCO router.
+  
+### Table Contents
+- [Requirement](#Requirement)
+- [Internet Protocol](#Internet-Protocol)
+- [Public and private ip address](#Public-and-private-ip-address)
+- [Private addressing](#Private-addressing)
+- [Network Address Translation](#Network-Address-Translation)
+- [Types of NAT](#Types-of-NAT)
+- [Configuring static NAT](#Configuring-static-NAT)
+- [Configuring dynamic NAT](#Configuring-dynamic-NAT)
+- [Configuring PAT](#Configuring-PAT)
+- [Conclusion](#Conclusion)
+- [Activity files](#Activity-files)
+- [Further reading](#Further-reading)
+
+### Requirement.
+This tutorial uses the CISCO packet tracer [Download packet tracer](https://www.netacad.com/portal/resources/packet-tracer).
+*NOTE To better understand this tutorial, the reader is expected to have a  foundational knowledge of IPV4 and the CISCO command-line interface (CLI)*
+
+### Internet Protocol
+According to [Wikipeida](https://en.wikipedia.org/wiki/IP_address), an Internet Protocol address (IP address) is a numerical label assigned to each device connected to a computer network that uses the Internet Protocol for communication.
+IP address currently have two version in used: Internet Protocol version 4 (IPv4) and Internet Protocol Version 6 (IPv6)
+IPv4 defines an IP address as a 32-bit number while IPv6 defines IP address as a 128-bit number
+### Public and private ip address
+All IPv4 addresses can be divided into two groups: public (global) and private (local) addresses.
+Public addresses are routable addresses that are used on the internet.
+These addresses allow users to access resources on a computer network located anywhere in the world.
+Private addresses are not routable  and no traffic can be sent to them or by them over the internet.
```

Comment:
Don't write them on separate lines. Combine them and make it as a para.

Generally, a para speaks about one idea. And, you break it when you introduce something else in the para.